src/yardstick/cli/config.py

```
@@ -125,15 +128,15 @@ def load(
         try:
             with open(cfg.profile_path, encoding="utf-8") as yaml_file:
                 profile = Profiles(yaml_decoder(yaml_file))
-        except:  # pylint: disable=bare-except
+        except FileNotFoundError:
```

Yay! Thanks for fixing the bare except statement.